packages/tuff-intelligence/src/runtime/decision-dispatcher.test.ts

```
@@ -0,0 +1,51 @@
+import { describe, expect, it } from 'vitest'
+import { CapabilityRegistry } from '../registry/capability-registry'
+import { DecisionDispatcher } from './decision-dispatcher'
+import type { AgentEnvelope } from '../protocol/envelope'
```

ESLint

[error] Expected "../protocol/envelope" (parent-type) to come before "./decision-dispatcher" (sibling).

(perfectionist/sort-imports)